src/types/zeroWallet.ts

```diff
@@ -26,3 +26,11 @@ export type SendGaslessTransactionType = string; // @TODO-update
 export type InitRelayerProps = InitBiconomyRelayerProps; // @TODO-update
 
 export type InitRelayerType = InitBiconomyRelayerType; // @TODO-update
+
+export type DatabaseConfig = {
```

Comment:
Where is this getting initialized? Where is the password being picked from?

Reply:
The user would need to pass these params to the constructor. 
So the user can store them in an .env file and then pass them from there

Reply:
OK